src/main/java/pl/wrapper/parking/facade/domain/ParkingController.java

```diff
@@ -1,13 +1,54 @@
 package pl.wrapper.parking.facade.domain;
 
+import static pl.wrapper.parking.infrastructure.error.HandleResult.handleResult;
+
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.RequestMapping;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import pl.wrapper.parking.facade.ParkingService;
+import pl.wrapper.parking.infrastructure.error.Result;
+import pl.wrapper.parking.pwrResponseHandler.dto.ParkingResponse;
+
+import java.util.List;
 
-@RequestMapping("v1")
 @RestController
 @RequiredArgsConstructor
+@Slf4j
 class ParkingController {
     private final ParkingService parkingService;
+
+    @GetMapping("/name")
+    public ResponseEntity<String> getParkingByName(@RequestParam String name, @RequestParam(required = false) Boolean opened) {
+        log.info("Received request: get parking by name: {}", name);
+        Result<ParkingResponse> result = parkingService.getByName(name, opened);
+        return handleResult(result, HttpStatus.OK, "v1/name");
+    }
+
+    @GetMapping("/id")
+    public ResponseEntity<String> getParkingById(@RequestParam Integer id, @RequestParam(required = false) Boolean opened) {
+        log.info("Received request: get parking by id: {}", id);
+        Result<ParkingResponse> result = parkingService.getById(id, opened);
+        return handleResult(result, HttpStatus.OK, "v1/name");
+    }
+
+    @GetMapping("/symbol")
+    public ResponseEntity<String> getParkingBySymbol(@RequestParam String symbol, @RequestParam(required = false) Boolean opened) {
+        log.info("Received request: get parking by symbol: {}", symbol);
+        Result<ParkingResponse> result = parkingService.getBySymbol(symbol, opened);
+        return handleResult(result, HttpStatus.OK, "v1/name");
+    }
+
+    @GetMapping("/params")
```

Comment:
Let's remove the url mapping completely here ( just @GetMapping). If no params supplied, all parking lots will be returned